.github/workflows/build_for_deployment.yaml

```diff
@@ -0,0 +1,50 @@
+name: mara-app
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  mara-app-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.7-dev']
```

Comment:
Lets go straight to 3.6/3.7 to 3.9 + 3.10-dev (if that's available, don't know)?

Reply:
I will update this to 3.6, 3.7, 3.8, 3.9, 3.10 if you want. Maybe, we can even move away from 3.6 if we only deploy 3.6?